src/lib/ndpi_content_match.c.inc

```diff
@@ -1703,6 +1703,68 @@ ndpi_protocol_match host_match[] =
 
    { "cachefly.com",           "CacheFly", NDPI_PROTOCOL_CACHEFLY, NDPI_PROTOCOL_CATEGORY_CLOUD, NDPI_PROTOCOL_ACCEPTABLE, NDPI_PROTOCOL_DEFAULT_LEVEL },
 
+   { "cyberreefportal.com",              "Cyberreef", NDPI_PROTOCOL_CYBERREEF, NDPI_PROTOCOL_CATEGORY_WEB, NDPI_PROTOCOL_SAFE, NDPI_PROTOCOL_DEFAULT_LEVEL },
+   { "test.cyberreefportal.com",         "Cyberreef", NDPI_PROTOCOL_CYBERREEF, NDPI_PROTOCOL_CATEGORY_WEB, NDPI_PROTOCOL_SAFE, NDPI_PROTOCOL_DEFAULT_LEVEL },
+   { "dev.cyberreefportal.com",          "Cyberreef", NDPI_PROTOCOL_CYBERREEF, NDPI_PROTOCOL_CATEGORY_WEB, NDPI_PROTOCOL_SAFE, NDPI_PROTOCOL_DEFAULT_LEVEL },
+   { "cyberreefportal-dev.azurewebsites.net",      "Cyberreef", NDPI_PROTOCOL_CYBERREEF, NDPI_PROTOCOL_CATEGORY_WEB, NDPI_PROTOCOL_SAFE, NDPI_PROTOCOL_DEFAULT_LEVEL },
+   { "cyberreefportal-test.azurewebsites.net",     "Cyberreef", NDPI_PROTOCOL_CYBERREEF, NDPI_PROTOCOL_CATEGORY_WEB, NDPI_PROTOCOL_SAFE, NDPI_PROTOCOL_DEFAULT_LEVEL },
+
+   { "tv.apple.com",                     "AppleTVPlus", NDPI_PROTOCOL_APPLETVPLUS, NDPI_PROTOCOL_CATEGORY_STREAMING, NDPI_PROTOCOL_FUN, NDPI_PROTOCOL_DEFAULT_LEVEL },
```

Comment:
Are "STREAMING" and "FUN" consistent with existing streaming services?  Disney+ comes to mind as one that the original product supported.

Reply:
All other streaming protos category was set to fun